pom.xml

```diff
@@ -61,18 +61,18 @@
 
     <organization>
         <name>JFrog</name>
-        <url>http://www.jfrog.org</url>
+        <url>https://jfrog.com</url>
     </organization>
 
     <issueManagement>
         <system>jira</system>
-        <url>http://issues.jfrog.org/jira/browse/HAP</url>
+        <url>https://www.jfrog.com/jira/browse/HAP</url>
```

Comment:
hmm, this smells 😅 
According to mozilla
> Yes, you need to choose one and stick with it. The choice of which one to have as your canonical location is yours, but if you choose one, stick with it. It will make your website appear more consistent to your users and to search engines. This includes always linking to the chosen domain (which shouldn't be hard if you're using relative URLs in your website) and always sharing links (by email/social networks, etc.) to the same domain.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Choosing_between_www_and_non-www_URLs